addons/costume-editor-shortcuts/addon.json

```
@@ -15,5 +15,10 @@
   ],
   "versionAdded": "1.41.0",
   "tags": ["editor", "costumeEditor", "recommended"],
+  "latestUpdate": {
+    "version": "1.42.0",
+    "isMajor": true,
+    "temporaryNotice": "The bug where the Ctrl key was ignored when reading shortcuts was fixed."
```

```suggestion
    "temporaryNotice": "The bug where shortcuts were still triggered while holding the Ctrl key was fixed."
```
maybe it should be reworded like this? something that somewhat more clearly says that this fixes Ctrl+A and stuff